scripts/test-local.sh

```diff
@@ -1,5 +1,11 @@
 #!/usr/bin/env bash
 
+glob=$*
+if [ "$glob" == "" ]; then
+  glob="test/local test/remote"
+fi
+
 ./scripts/gen_keys.js
 ./scripts/check-i18n.js
-./scripts/tap-coverage.js test/local test/remote 2>/dev/null
+./scripts/tap-coverage.js $glob 2>/dev/null
+grunt
```

Comment:
I didn't know what the default did myself, I just happily moved `grunt` from the package.json to this line. I like not relying on the default as well.